### ISSUE TITLE
Doesn't run with iPhone plugged in on Mac

### ISSUE THREAD

**Issue by Feliix42** [hephaest0s/usbkill#61]
Hello,

as I tried to run **usbkill** with my iPhone and a mouse (which I was able to eliminate as source of the problem) attached to the USB ports, I received this error:

```
             _     _     _ _ _  
            | |   | |   (_) | | 
  _   _  ___| |__ | |  _ _| | | 
 | | | |/___)  _ \| |_/ ) | | | 
 | |_| |___ | |_) )  _ (| | | | 
 |____/(___/|____/|_| \_)_|\_)_)

Traceback (most recent call last):
  File "usbkill.py", line 376, in <module>
    loop(settings)
  File "usbkill.py", line 252, in loop
    start_devices = lsusb()
  File "usbkill.py", line 167, in lsusb
    return lsusb_darwin()
  File "usbkill.py", line 160, in lsusb_darwin
    check_inside(result, devices)
  File "usbkill.py", line 153, in check_inside
    check_inside(result_deep, devices)
  File "usbkill.py", line 145, in check_inside
    devices.append(DEVICE_RE[1].findall(result["vendor_id"])[0] + ':' + DEVICE_RE[1].findall(result["product_id"])[0])
IndexError: list index out of range
```

After I unplugged the phone, the script started properly. 
I use a MacBook Air Mid 2013 with OS X 10.10.3.

And - when I terminate the program with Ctrl + C, this error is thrown:
```
Traceback (most recent call last):
  File "usbkill.py", line 376, in <module>
    loop(settings)
  File "usbkill.py", line 280, in loop
    sleep(settings['sleep_time'])
  File "usbkill.py", line 284, in exit_handler
    log("[INFO] Exiting because exit signal was received")
TypeError: log() takes exactly 2 arguments (1 given)
```

**Comment by pwnsdx** [comment]
Hello, thanks you for reporting this issue!

I will look into it.

S.

**Comment by hephaest0s** [comment]
Thanks for reporting these two issues.

The Ctrl + C exit error is now resolved in version 1.0-rc.2
